src/components/AboutUs/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import MeetTheTeam from "../MeetTheTeam";
+import aboutus from "./images/aboutus.png";
+import "./index.css";
+
+const AboutUs = () => {
+  return (
+    <div>
+      <Container>
+        <Row>
+          <Col className="aboutUs">
+            <h3>About NO BARRIERS</h3>
+            <p>
```

Comment:
why arent we using the translation for this part ?

Reply:
Oh, you're right. I will update it and the others.

Reply:
> why arent we using the translation for this part ?

Oh, you're right. I will update it and the others.